charts/scalardb/templates/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
```

Comment:
I think we need to add a `PrometheusRule` and `ServiceMonitor` for prometheus since scalardb-server now exposes Prometheus mertics.
https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardl/templates/ledger/prometheusrules.yaml
https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardl/templates/envoy/servicemonitor.yaml